pages/help.markdown

```diff
@@ -252,7 +258,7 @@ comments: []
 		You can simply add graphics or images to your screenshot by dragging and dropping an image<br />
 		file into the editor window. You can also insert screenshots of other windows by selecting<br />
 		<em>Insert window</em> from the <em>Edit</em> menu. A list of all open windows appears,<br />
-		allowing you to select one for insertion.
+		allowing you to select one for insertion. Insert an image from the clipboard by <kbd>Ctrl</kbd> + <kbd>v</kbd> works as well.
```

Comment:
Actually what most people don't know, is that one can also drag & drop from a file explorer or a browser. 🤷